MDANSE/Src/MDANSE/Framework/Jobs/RotationAutocorrelation.py

```diff
@@ -0,0 +1,214 @@
+#    This file is part of MDANSE.
+#
+#    MDANSE is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import collections
+
+import numpy as np
+from scipy.signal import correlate
+from scipy.spatial.transform import Rotation
+
+from MDANSE.Framework.Jobs.IJob import IJob
+from MDANSE.Mathematics.Geometry import center_of_mass, moment_of_inertia
+
+
+class RotationAutocorrelation(IJob):
+    """Calculates the rotation angle correlation of molecules.
+
+    For each molecule, the Kabsch algorithm is used to determine the
+    rotation angles relative to the original position at each time step.
+
+    """
+
+    enabled = False
+
+    label = "Rotation Autocorrelation"
+
+    category = (
+        "Analysis",
+        "Dynamics",
+    )
+
+    ancestor = ["hdf_trajectory", "molecular_viewer"]
+
+    settings = collections.OrderedDict()
+    settings["trajectory"] = ("HDFTrajectoryConfigurator", {})
+    settings["frames"] = (
+        "CorrelationFramesConfigurator",
+        {"dependencies": {"trajectory": "trajectory"}},
+    )
+    settings["molecule"] = (
+        "MoleculeSelectionConfigurator",
+        {
+            "label": "molecule name",
+            "default": "",
+            "dependencies": {"trajectory": "trajectory"},
+        },
+    )
+    settings["atom_selection"] = (
+        "AtomSelectionConfigurator",
+        {"dependencies": {"trajectory": "trajectory"}},
+    )
+    settings["output_files"] = ("OutputFilesConfigurator", {})
+    settings["running_mode"] = ("RunningModeConfigurator", {})
+
+    def initialize(self):
+        """Initialize the input parameters and analysis self variables."""
+        super().initialize()
+
+        molecules = self.configuration["trajectory"][
+            "instance"
+        ].chemical_system._clusters[self.configuration["molecule"]["value"]]
+
+        self._indices = set(
+            idx
+            for idxs in self._configuration["atom_selection"]["indices"]
+            for idx in idxs
+        )
+
+        self.valid_molecules = [mol for mol in molecules if set(mol) <= self._indices]
```

Comment:
```suggestion
        self.valid_molecules = [mol for mol in molecules if self._indices.issuperset(mol)]
```
May be cleaner than casting to `set`?